### DISCUSSION TITLE
How to exclude a specific lib from being linked static?

### DISCUSSION THREAD

**Discussion by jajabu33** [microsoft/vcpkg#33419] [Q&A]
I'm linking some libraries with the triplet windows-static:

![image](https://github.com/microsoft/vcpkg/assets/106472796/2df28e0f-a2d4-4433-8cb4-ab245f24714e)

I'm trying to exclude a specific library (`openssl`) from being linked as static while keeping all others static.

While searching around I found this [answer][1]:
> If you want to locally override the default linkage for a particular port, you would put the override into a custom (overlay) triplet file.

But i couldn't figure out how to do it.

Also, this [answer][2], but the links in the Microsoft GitHub throw `404 - page not found`.

  [1]: https://github.com/microsoft/vcpkg/issues/17939#issuecomment-841969081
  [2]: https://stackoverflow.com/a/65769727/22456056

**Comment by Neumann-A** [comment]
Custom triplet:

https://learn.microsoft.com/en-us/vcpkg/users/triplets#per-port-customization

Using custom triplets: 

https://learn.microsoft.com/en-us/vcpkg/users/examples/overlay-triplets-linux-dynamic?source=recommendations

In the VS Integration you probably need to fill in `Additional Options` with the correct CLI arguments

**Reply by jajabu33** [reply]
What options? i read the doc you linked but i still did not understand how to configure it

https://github.com/Intelight/vcpkg/blob/master/triplets/x86-windows-mixed.cmake

I need to create a x64-windows-mixed.cmake in my project folder? this is so confuse

**Comment by jajabu33** [comment]
bump
